src/lightning_app/core/app.py

```diff
@@ -18,12 +18,14 @@
     DEBUG_ENABLED,
     FLOW_DURATION_SAMPLES,
     FLOW_DURATION_THRESHOLD,
+    FRONTEND_DIR,
     STATE_ACCUMULATE_WAIT,
 )
 from lightning_app.core.queues import BaseQueue, SingleProcessQueue
 from lightning_app.frontend import Frontend
 from lightning_app.storage import Drive, Path
 from lightning_app.storage.path import storage_root_dir
+from lightning_app.utilities import add_meta
```

Comment:
Maybe `add_meta` can be renamed to something else?

Reply:
frontend.py